Bonobo.Git.Server/Git/Models/BatchApiResponse.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Xml;
+
+namespace Bonobo.Git.Server.Git.Models
```

Comment:
Not a fan of the term `models` for objects that can be described as part of the interface. I think these objects deserve the most direct and shortest namespacing. While I will also sometimes include a couple of classes under one file, this seems like a lot to put in one place.

Reply:
After a discussion, we agreed that these are models.  There are only 2 classes, but it is a lot to put in one place.  Even though the second class is only 4 lines, the larger class might be more clear if I break them into their own files anyway so I will do that.